### ISSUE TITLE
Breaking change with uri-ssh_git

### ISSUE THREAD

**Issue by sanemat** [packsaddle/ruby-git_clone_url#6]
uri-ssh_git v2.0 will distinguish absolute path and relative path.
see https://github.com/packsaddle/ruby-uri-ssh_git/issues/14

v1 behavior: `#parse` always returns path which start with '/', and `Generic#to_s` always returns path's first '/' stripped.
v2 behavior: `#parse` returns path which depends on input (absolute path or relative path)

```
url = URI::SshGit.parse('git@github.com:packsaddle/ruby-uri-ssh_git.git')
# => #<URI::SshGit::Generic git@github.com:packsaddle/ruby-uri-ssh_git.git>

# http://docs.ruby-lang.org/ja/2.2.0/class/URI=3a=3aGeneric.html
# :scheme ,:userinfo, :host, :port, :registry, :path, :opaque, :query, :fragment
url.scheme #=> nil
url.userinfo #=> 'git'
url.user #=> 'git'
url.password #=> nil
url.host #=> 'github.com'
url.port #=> nil
url.registry #=> nil
url.path #=> 'packsaddle/ruby-uri-ssh_git.git'
url.opaque #=> nil
url.query #=> nil
url.fragment #=> nil
```

**Comment by sanemat** [comment]
- https://github.com/packsaddle/ruby-git_httpsable-push
- https://github.com/packsaddle/ruby-saddler-reporter-support-git
- https://github.com/packsaddle/ruby-pull_request-create
- https://github.com/opskeleton/opskeleton
- https://github.com/narkisr/puppet-base-env
- https://github.com/blue-systems/pangea-tooling

**Comment by sanemat** [comment]
https://rubygems.org/gems/git_clone_url v1.0.1 is published.
git_clone_url v1.x uses uri-ssh_git v1.x.

**Comment by sanemat** [comment]
https://rubygems.org/gems/git_httpsable-push v0.1.1 is published.
httpsable-push v0.1.x uses git_clone_url v1.x.

**Comment by sanemat** [comment]
https://github.com/packsaddle/ruby-uri-ssh_git/issues/14

**Comment by sanemat** [comment]
I recommend this workaround:

``` diff
-  spec.add_runtime_dependency 'git_clone_url'
+  spec.add_runtime_dependency 'git_clone_url', '>= 1.0', '< 2.0'
```

**Comment by sanemat** [comment]
I released `git_clone_url v2.0.0.pre.1`. :basketball:

**Comment by sanemat** [comment]
I released `git_clone_url v2.0.0`. :factory:

**Comment by sanemat** [comment]
Fixed!
